ga4gh/protocol.py

```diff
@@ -11,22 +11,21 @@
 
 import google.protobuf.json_format as json_format
 import google.protobuf.message as message
+import google.protobuf.struct_pb2 as struct_pb2
```

Comment:
@calbach, never mind I found the `struct_pb2.py` file in the Protobuf Python APIs in the following tar.gz file, which seems to be auto-generated:

https://pypi.python.org/packages/source/p/protobuf/protobuf-3.0.0b2.tar.gz#md5=f0d3bd2394345a9af4a277cd0302ae83

Some of this might need to be documented as more folks joining GA4GH will work with Protobuf.

~p